ipatests/test_integration/test_replica_promotion.py

```diff
@@ -1300,3 +1300,65 @@ def test_kra_hidden_temp(self):
             self.replicas[0].hostname, '--state=hidden'
         ])
         assert result.returncode == 0
+
+
+class TestReplicaConn(IntegrationTest):
+    ad_user_login = None
+    topology = "line"
+    num_clients = 1
+    num_ad_domains = 1
+
+    @classmethod
+    def install(cls, mh):
+        super().install(mh)
+
+        cls.replica = cls.clients[0]
```

Comment:
Since you're using a replica, why not define num_replicas = 1 and num_clients = 0 instead and cls.replica = cls_replicas[0]?
If you don't want the replica to be installed by the setup, you just need to remove the line `topology = "line"`